src/azul/collections.py

```diff
@@ -62,35 +64,44 @@ def explode_dict(d: Mapping[K, Union[V, List[V], Set[V], Tuple[V]]]) -> Iterable
         yield dict(zip(d.keys(), t))
 
 
-def none_safe_key(v):
+def none_safe_key(none_last: bool = True) -> Callable[[Any], Any]:
     """
-    A sort key that handles None values.
+    A sort key factory that handles None values.
 
     >>> sorted([1, None, 2])
     Traceback (most recent call last):
     ...
     TypeError: '<' not supported between instances of 'NoneType' and 'int'
 
-    >>> sorted([1, None, 2], key=none_safe_key)
+    >>> sorted([1, None, 2], key=none_safe_key())
+    [1, 2, None]
+
+    >>> sorted([1, None, 2], key=none_safe_key(none_last=False))
     [None, 1, 2]
     """
-    return v is not None, v
+    if none_last:
+        return lambda v: (v is None, v)
+    else:
+        return lambda v: (v is not None, v)
 
 
-def none_safe_tuple_key(t):
+def none_safe_tuple_key(none_last: bool = True) -> Callable[[Any], Any]:
```

Comment:
Wouldn't `False` be a better default value for `none_last`? In older Pythons with total ordering on all values, the None value would come first naturally.